src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -126,8 +141,15 @@ public void canCreateAnInstance()
     TimeoutException {
 
     UUID id = UUID.randomUUID();
+    String[] natureOfContentIds = Stream.of(
```

Comment:
Whilst this is elegant, I find it harder to read than something like

journalContentTermId = createNatureOfContentTerm("journal_test");
bookContentTermId = createNatureOfContentTerm("book_test");

And then construct the list from those.

Reply:
@marcjohnson-kint, done